bin/helpers/generate_features.py

```diff
@@ -0,0 +1,60 @@
+"""Generate features for equity bars data from raw data in S3.
```

Comment:
Keeps sneaking back in but this file can be removed. The other changes are fine.